app/domain/authentication/authn_jwt/restriction_validation/validate_restrictions_one_to_one.rb

```diff
@@ -4,21 +4,34 @@ module RestrictionValidation
       # This class is responsible for retrieving the correct value from the JWT token
       # of the requested attribute.
       class ValidateRestrictionsOneToOne
-        def initialize(decoded_token:)
+        def initialize(decoded_token:, mapping_claims:)
```

Comment:
i think `mapped_claims_dictionary` or `mapped_claims` is more clear then `mapping_claims`
Also in the UTs you called the parameter mapped_claims

Reply:
WDYT of adding a default empty value to mapped_claims in the initialize function??

Reply:
I think that empty value there can lead to non intentional bug.

Reply:
I'll change it to mapped_claims